src/test/java/eu/elixir/ega/ebi/it/res/FileControllerIntgTest.java

```diff
@@ -0,0 +1,109 @@
+package eu.elixir.ega.ebi.it.res;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigInteger;
+import java.security.DigestInputStream;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.junit.Test;
+
+import com.google.common.io.CountingInputStream;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+/**
+ * Class to do the integration test for FileController in ega-data-api-key.
+ * 
+ * @author amohan
+ *
+ */
+public class FileControllerIntgTest extends ResBase {
+
+    private final static String FILE_PATH = "/file/archive/";
+    private OkHttpClient client;
+
+    /**
+     * Verify the api call /file/archive/{id} and check status is
+     * {@link org.apache.http.HttpStatus#SC_OK}. Also checks the response body
+     * should not be null.
+     */
+    @Test
+    public void testGetArchiveFile() {
+
+        String url = baseURI + ":" + port + FILE_PATH + fileId + "?destinationFormat=plain";
+        requestRequest = new Request.Builder().url(url).build();
+
+        client = SSLUtilities.getUnsafeOkHttpClient();
+
+        // Set up Digest
+        MessageDigest md = null;
+        try {
+            md = MessageDigest.getInstance("MD5");
+        } catch (NoSuchAlgorithmException ex) {
+        }

```

Comment:
I am not sure about how the restAssured works, but I would add the exception to the method signature instead of ignoring it.

Reply:
Added throws exception